uint8 NONE = 0
uint8 FRONT = 1
uint8 FRONT_LEFT = 2
uint8 LEFT = 3
uint8 REAR_LEFT = 4
uint8 REAR = 5
uint8 REAR_RIGHT = 6
uint8 RIGHT = 7
uint8 FRONT_RIGHT = 8

uint8 bump_direction
